tests/modelgauge_tests/test_safe.py

```diff
@@ -256,10 +251,10 @@ def safe_test(self):
     def test_make_test_items_skips_out_of_scope_prompts(self, tmpdir, safe_test):
         csv_header = ["prompt_uid", "prompt_text", "hazard", "locale", "persona"]
         rows = [
-            ["1", "a", self.hazard, Locale.EN_US, SafePersonasVersion1.NORMAL],
-            ["2", "a", self.hazard, Locale.EN_US, SafePersonasVersion1.SKILLED],
-            ["3", "a", self.hazard, Locale.FR_FR, SafePersonasVersion1.NORMAL],
-            ["4", "a", self.hazard, Locale.FR_FR, SafePersonasVersion1.SKILLED],
+            ["1", "a", self.hazard, "en_US", "normal"],
```

Comment:
Mixed case in the locale keys makes me nervous. We know those strings are easy to get wrong (cf. "HI_HI" from a few weeks ago) and I wonder if mixed case will make that worse. 

At the same time, xx_YY is the standard case for locale language strings, so that's as good a reason as any to use xx_YY.

(I don't have an alternative proposal, I'm just bringing it up as a concern)

Reply:
Yeah, personally I'd rather use all lower case, as I have in the CLI options and the UIDs. I considered making Locale accept any case, but decided this PR was huge anyhow.